app/Http/Controllers/Api/PostsController.php

```diff
@@ -121,6 +116,6 @@ public function reviews(Request $request)
         $meta = [];
 
         // @TODO: we'll need to change the transformer here depending on what type of post.
-        return $this->collection($reviewedPosts, $code, $meta, $this->photoTransformer);
+        return $this->item($reviewedPost, $code, $meta, $this->photoTransformer);
```

Comment:
@sbsmith86 I actually don't think we even need this here? I don't think it's being used!